app/views/analytics/facilities/show.csv.erb

```diff
@@ -18,11 +18,11 @@
 <% @cohort_analytics.each_with_index do |(report_dates, analytics), index| %>
   <%
     cohort_start, report_start = report_dates
-    registered = analytics[:registered]
-    followed_up = analytics[:followed_up]
-    defaulted = analytics[:defaulted]
-    controlled = analytics[:controlled]
-    uncontrolled = analytics[:uncontrolled]
+    registered = analytics.dig(:registered, :total) || analytics[:registered]
+    followed_up = analytics.dig(:followed_up, :total) || analytics[:followed_up]
+    defaulted = analytics.dig(:defaulted, :total) || analytics[:defaulted]
+    controlled = analytics.dig(:controlled, :total) || analytics[:controlled]
+    uncontrolled = analytics.dig(:uncontrolled, :total) || analytics[:uncontrolled]
```

Comment:
We leave the old `analytics[:registered]` method in here to ensure that the code still works with the older data structure that'll be in cache when we deploy this change. 48 hours after we deploy this change, we can clean up the second clause.